test/e2e/test.utils.js

```diff
@@ -12,7 +12,8 @@ export const accountAddresses = {
 };
 
 export const webWalletURL = 'https://wallet.agoric.app/';
-export const MINUTE_MS = 60000;
+export const MINUTE_MS = 1 * 60 * 1000;
+export const TWO_MINUTE_MS = 2 * 60 * 1000;
```

Comment:
minute_ms was created to create give a name to the 60k constant value.
2 * MINUTE_MS is clear enough to indicate that it means 2 mins

i dont think we should be creating a seperate variable for every minute

Reply:
Updated